### PR TITLE
Cleaned up conduct page

### LINKED ISSUE #468: Styles are off on the conduct page
Not sure of the status of this page, but if it's a keeper the styles need to get mopped up. If it's deprecated we should look to cut it. 

https://jupyter.org/conduct/

Screenshot from 2021-12-04 10-42-27

### PR DESCRIPTION
**Pull request by palewire**
Fixes #468

### CODE REVIEW

**Comment by krassowski** [comment]
Before:

Screenshot from 2021-12-05 18-31-40

After:

Screenshot from 2021-12-05 18-32-15

**Review by krassowski — APPROVED** [review]
Thanks, I believe it does make it easier to read by using a constrained layout.

File: conduct.md [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
+In case of difference, the version on the [Jupyter/Governance GitHub
 Repository](https://github.com/jupyter/governance) is the canonical one, and
 the one present here should be updated accordingly.
 
-
-
-
 ------------------------
 
-# Project Jupyter Code of Conduct
```
Comment (krassowski) [inline conduct.md:18]: I would keep it, because it separates the explanation above from the actual code of conduct.

Comment (palewire) [reply conduct.md:18]: Done. It should be back in.

**Review by isabela-pf — APPROVED** [review]
This is such a great fix, thank you!

**Review by Carreau — APPROVED** [review]

**Pull request merged**